src/lambda_handler.py

```diff
@@ -1,89 +1,231 @@
 """Simple AWS Lambda handler to verify functionality."""
```

Comment:
The docstring for this file should be updated from the skeleton:
```suggestion
"""AWS Lambda handler to sync NVD data with a MongoDB."""
```

Reply:
@dav3r Addressed this here https://github.com/cisagov/cyhy-nvdsync-lambda/pull/2/commits/7169a0a03d4a1758e3fdc9f8a92490175a18c8c3

Reply:
This convo should've been automatically resolved when you committed my suggestion.  Not sure what happened here.

Reply:
@dav3r I was thinking the original commenter needed to approve the changes. So I had unresolved it but I will just leave it resolved in the future. Just wanted to make sure I followed processes correctly

Reply:
Your thinking was correct- this is the only exception to our "resolve conversations" rule.  When a suggestion is accepted, GitHub automatically resolves the conversation on behalf of the original commenter since you are committing exactly what the original commenter suggested.